plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -323,7 +318,7 @@ else if (table.getTableType().equals(EXTERNAL_TABLE.name())) {
                 throw new TrinoException(HIVE_METASTORE_ERROR, "Could not validate external location", e);
             }
         }
-        else if (!table.getTableType().equals(MATERIALIZED_VIEW.name())) {
+        else if (!(table.getTableType().equals(MANAGED_TABLE.name()) || table.getTableType().equals(MATERIALIZED_VIEW.name()))) {
```

Comment:
I'd reformat this check:
```java
else if (!table.getTableType.equals(MANAGED_TABLE.name())
        && !table.getTableType.equals(MATERIALIZED_VIEW.name())) {
```
I think `!a && !b` is a bit clearer than `!(a || b)` when it comes to excluding multiple values, and the split line lets it fit on a screen at a reasonable width.